.ado/xcconfig/publish_overrides_ios_device.xcconfig

```diff
@@ -2,3 +2,4 @@
 
 // Explicitly build arm64e for iOS device builds in case our clients need it
 ARCHS = $(ARCHS_STANDARD) arm64e
+ENABLE_ADDRESS_SANITIZER = NO
```

Comment:
This only affects builds produced by the ADO pipeline, right? Not local device builds? And does this affect the Github build pipe?

Reply:
it shouldn't it is only for ado pipline.